ext/mysqlnd/mysqlnd_debug.c

```diff
@@ -543,7 +543,7 @@ MYSQLND_METHOD(mysqlnd_debug, set_mode)(MYSQLND_DEBUG * self, const char * const
 				state = PARSER_WAIT_COLON;
 				break;
 			case ':':
-#if 0
+#if PHP_DEBUG
```

Comment:
ZEND_DEBUG/PHP_DEBUG should not change observable behavior. So either this should stay as-is or the `#if` needs to be dropped completely.

Reply:
I'll drop it then, as to IMHO, I don't see why this is being disabled currently